auth-manager/src/main/java/com/qpay/authmanager/service/jwt/impl/JwtServiceImpl.java

```diff
@@ -21,14 +23,17 @@ public class JwtServiceImpl implements JwtService {
     private String jwtSigningKey;
 
     @Override
-    public String generateToken(final String email) {
-        return createToken(Map.of(), email);
+    public String generateToken(final String email, final long userId, final UserType userType) {
+        final Map<String, Object> claims = new HashMap<>();
+        claims.put("email", email);
+        claims.put("userId", userId);
+        claims.put("userType", userType);
+        return createToken(claims);
     }
 
-    private String createToken(final Map<String, Object> claims, final String email) {
+    private String createToken(final Map<String, Object> claims) {
         return Jwts.builder()
                 .setClaims(claims)
-                .setSubject(email)
```

Comment:
are you sure you don't need it ?
I have no idea

Reply:
i put it in `claims` and then checked in JWT Io how it looks and everithing normal, except that now this field `email` not `sub`